### PR TITLE
[script.videolinks] 1.0.1

### PR DESCRIPTION
**Pull request by the-black-eagle**
### Description
Although the python artist scraper now fetches video links from theaudiodb, this standalone version can fetch the links for just a single artist ( or all artists) without updating anything else.  It's therefore possible using this addon to add links at theaudiodb and then update Kodi with them, without affecting anything else.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by basrieter** [comment]
Could you please fix your conflicts (most likely by rebasing)?

**Comment by basrieter** [comment]
Please rebase to resolve the conflicts.

File: script.videolinks/default.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```
Comment (romanvm) [inline script.videolinks/default.py:1]: Not needed in Py 3

**Review by romanvm — DISMISSED** [review]

File: targets.cfg
Comment (basrieter) [inline targets.cfg:1]: This file should not be in your PR. Please exclude it.

**Review by basrieter — CHANGES_REQUESTED** [review]

**Comment by romanvm** [comment]
Please resolve merge conflicts

**Comment by basrieter** [comment]
*ping* @the-black-eagle

**Comment by basrieter** [comment]
Closing due to non-responsiveness.

**Pull request closed without merging**